### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by RadValentin**
## Motivation (required)

Some fixes for various grammatical and styling inconsistencies found while browsing the docs. Mostly low-level stuff. There are definitely other aspects that could be improved, will list my findings below (for feedback purposes) and leave it for a future PR.

- Online docs are outdated, contain references to deprecated functionality, opened issue in #382
- Couldn't find any info on how to generate the docs locally
- No clear separation in docs between VR APIs - `Animated` and RN APIs - `LayoutAnimation`, it's not clear where I should import something from

## Test Plan

Not needed

### CODE REVIEW

File: Libraries/VRLayers/CylindricalPanel.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ const requireNativeComponent = require('requireNativeComponent');
  * </CylindricalPanel>
  * ```
  *
- * The width and height of the cylinder must be specified, and the width must also correspond to
- * the arc that the cylinder covers. The default Density is 4680px, which has been empirically
+ * The width and height of the cylinder must be specified, and the width must also corresponds to
```
Comment (matthamil) [inline Libraries/VRLayers/CylindricalPanel.js]: I think "correspond" was correct initially

Comment (RadValentin) [reply Libraries/VRLayers/CylindricalPanel.js]: Good catch, fixed it in https://github.com/facebook/react-vr/pull/385/commits/d300f994444e8d91b28840413ffa62cd8d05ebea Thanks!

**Comment by RadValentin** [comment]
@mikearmstrong001 could you help with reviewing this PR so that it can be merged?

**Comment by RadValentin** [comment]
It would be awesome if we could get this PR merged or at least get some feedback on how it can be improved. @andrewimm can you help me with this?

File: docs/3DCoordinatesAndTransforms.md [hunk 2]
```diff
@@ -46,11 +52,21 @@ Rotation units are in degrees.
...
+      {scale: 0.1},
+      {rotateY: this.state.rotation},
+      {rotateX: -90}
+    ]
+  }}
+  source={{
+    url: './creature/',
+    mesh: 'creature.obj',
+    mtl: 'creature.mtl',
+    lit: true
```
Comment (andrewimm) [inline docs/3DCoordinatesAndTransforms.md]: This code is outdated. It should be
```
source={{
  obj: asset('creature.obj'),
  mtl: asset('creature.mtl'),
}}
```

File: docs/Fonts.md [hunk 3]
```diff
@@ -83,7 +83,7 @@ It is also possible to generate your own font set textures using the fontue tool
 The base command line for the EFIGS font is:
 
 ```
-OculusSans-Medium.otf efigs\efigs -co -0.01 -ts 1.0 -hpad 128 -vpad 128 -sdf 256 1024 1024 -cf data\strings\values\strings.xml -cf data\strings\values-de\strings.xml -cf data\strings\values-en-rGB\strings.xml -cf data\strings\values-es\strings.xml -cf data\strings\values-es-rES\strings.xml -cf data\strings\values-fr\strings.xml -cf data\strings\values-it\strings.xml -cf data\EFIGS_extended.txt
+OculusSans-Medium.otf efigs/efigs -co -0.01 -ts 1.0 -hpad 128 -vpad 128 -sdf 256 -1 -1 -ur 0x0010 0x00ff -ur 0x0370 0x03FF -ur 0x0100 0x017F -ur 0x0180 0x024F
```
Comment (andrewimm) [inline docs/Fonts.md]: I'm not familiar with the fontue tool. Will need @mikearmstrong001 to take a look at this

**Comment by RadValentin** [comment]
Fixed review comment, waiting for @mikearmstrong001 to address changes made to Fontue example.

**Comment by RadValentin** [comment]
@mikearmstrong001 Pinging you again, I really want to see this PR merged, please take a quick look, it shouldn't take more than a couple of minutes.

do it

**Comment by mmontag** [comment]
Possible to merge this yet?

**Comment by andrewimm** [comment]
@mmontag please rebase past e0ddaca952e6f4d9b0a834d78faebf964fd370da, and we'll be able to import it into our internal repo and accept it.

**Comment by RadValentin** [comment]
@andrewimm Can you help me get this PR merged?

**Pull request closed without merging**